src/work-allocation/utils/work-allocation-utils.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { handleFatalErrors, REDIRECTS, treatAsFatal, WILDCARD_SERVICE_DOWN } from './work-allocation-utils';
+
+describe('WorkAllocationUtils', () => {
+    const mockRouter = {
```

Comment:
Looks like everything is a 4 space indent. Should be 2.

Reply:
Corrected the indenting